code/modules/surgery/organ_manipulation.dm

```diff
@@ -69,32 +101,6 @@
 		/datum/surgery_step/mechanic_close,
 	)
 
-/datum/surgery/organ_manipulation/mechanic/next_step(mob/living/user, modifiers)
-	if(location != user.zone_selected)
-		return FALSE
-	if(user.combat_mode)
-		return FALSE
-	if(step_in_progress)
-		return TRUE
-
-	var/try_to_fail = FALSE
-	if(LAZYACCESS(modifiers, RIGHT_CLICK))
-		try_to_fail = TRUE
-
-	var/datum/surgery_step/step = get_surgery_step()
-	if(isnull(step))
-		return FALSE
-	var/obj/item/tool = user.get_active_held_item()
-	if(tool)
-		tool = tool.get_proxy_attacker_for(target, user)
```

Comment:
All these are important stuff for the borg omnitool to work. Did you test if the normal organ manipulation surgery works along with borgs?

Reply:
Oh, at a glance, it looked just like standard, but I'll give it another look.

Reply:
I believe this entire section above here and the two `try_op(...)` lines below are the same as the parent `next_step(...)`:
https://github.com/tgstation/tgstation/blob/7a7443090b8c23a44048fbbfe2fb3ca7cf908792/code/modules/surgery/surgery.dm#L111-L130
However, it looks like the tool check below is different from the parent:
https://github.com/tgstation/tgstation/blob/7a7443090b8c23a44048fbbfe2fb3ca7cf908792/code/modules/surgery/surgery.dm#L131-L135
Where it checks for whether the tool has a tool behaviour instead of being a surgical tool

I don't think removing this would cause issues with borg behaviour, as the parent still has the exact same proxy attacker code as shown above
But! As the comment below says, `//Mechanic organ manipulation isn't done with just surgery tools`, in comparison to the comment for the parallel check on the parent, `//Just because you used the wrong tool it doesn't mean you meant to whack the patient with it`, I assume this code was here to stop you from whacking people in the chest when using the wrong tools during mechanical organ manipulation surgery
So it may be good to look into that

Reply:
Right, so the solution is to make a new proc (or var, flag w/e) that checks if we shouldn't wack the person with the tool we just used if it has the surgery tool flag for standard surgeries or if it has a tool behaviour for mechanical organ manipulation. Thanks for the help.